src/data_parser.py

```diff
@@ -83,12 +84,10 @@ def get_course_details(course_id: str, term: int, year: int, offer: int, max_ret
                 and details["COURSE_TITLE"]
                 and details["TERM_DESCR"]
             ):
-                # TODO: Logger
-                # print(f"Course Details Not Found")
+                logger.debug("Course Details Not Found")
```

Comment:
This should be an error. Also output some course info with it as well, like `COURSE_TITLE` and `TERM_DESCR` since it will be in a seperate file to debug messages.